src/main/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Excavator extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Excavator start working");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("Excavator stop working");

```

Comment:
Similarly, the stop message 'Excavator stop working' should be more informative and indicate clearly that it's an Excavator that has stopped working. A better message would be 'Excavator has stopped working'.